doc/admin/performance_tuning.md

```diff
@@ -56,53 +70,57 @@ i2048 0      2Kb iovec Input only
 ```
 
 For a full description of `self_test` usage, run:
+
 ```bash
-$ ../bin/self_test --help
+$ ./bin/self_test --help
 ```
 
 **To start test_group server:**
 ```bash
 $ /usr/lib64/openmpi3/bin/orterun --mca btl self,tcp -N 1 \
-  --hostfile srvhostfile --output-filename testLogs/ \
-  -x D_LOG_FILE=testLogs/test_group_srv.log -x D_LOG_FILE_APPEND_PID=1 -x D_LOG_MASK=WARN \
-  -x CRT_PHY_ADDR_STR=ofi+sockets -x OFI_INTERFACE=eth0 \
-  -x CRT_CTX_SHARE_ADDR=0 -x CRT_CTX_NUM=16 \
-  ../bin/crt_launch -e tests/test_group_np_srv --name self_test_srv_grp --cfg_path=. &
+  --hostfile server_hostfile --output-filename testLogs/ \
+  -x D_LOG_FILE=testLogs/test_group_srv.log -x D_LOG_FILE_APPEND_PID=1 \
+  -x D_LOG_MASK=WARN -x CRT_PHY_ADDR_STR=ofi+sockets -x OFI_INTERFACE=eth0 \
+  -x CRT_CTX_SHARE_ADDR=0 -x CRT_CTX_NUM=16 -x CRT_ATTACH_INFO_PATH=. \
+  ./bin/crt_launch -e lib/daos/TESTING/tests/test_group_np_srv \
+  --name self_test_srv_grp --cfg_path=. &
 ```
```

Comment:
I would advise to provide instruction on how to run it against the DAOS server and assuming that the DAOS server is already running.